listenbrainz/alpha_importer.py

```diff
@@ -202,6 +209,11 @@ def import_from_alpha(username, token):
         except HTTPError as e:
             time.sleep(config.IMPORTER_DELAY)
 
+        except BetaAuthorizationException:
+            logger.error('Invalid authorization token for user %s', username)
+            logger.info('Stopping alpha import for user %s', username)
```

Comment:
Does this effectively remove the user from the alpha queue?

Reply:
yep, after this it returns back to the main loop. https://github.com/metabrainz/listenbrainz-server/pull/263/files#diff-1adf6645434862a9aa0554be577a691aR215